src/openrct2/ride/RideData.h

```diff
@@ -187,6 +187,9 @@ struct RideTypeDescriptor
     RideColourPreview ColourPreview;
     RideColourKey ColourKey;
 
+    // json name lookup
+    const char* Name;
```

Comment:
What's the difference between string_view and const char* I'm not too familiar with it.

Reply:
Not a lot, `std::string_view` is kind of a class with a `const char*` + size of string, but it does feel a bit more C++zy. @ZehMatt can probably explain it better.

Reply:
string_view is a good choice here since the string is not mutable, the benefit of string_view is that it knows the size of the string, so operations like searching can be simplified a bit by comparing length first.